### ISSUE TITLE
Space doesn't work

### ISSUE THREAD

**Issue by learn2reid** [alfredapp/placeholder-images-workflow#1]
### Workflow version

2024.2

### Alfred version

5.5.1

### macOS version

13.7.2

### Debugger output

```alfred_debugger
[15:23:25.885] Placeholder Images[Keyword] Processing complete
[15:23:25.889] Placeholder Images[Keyword] Passing output '300 300' to Split Arg
[15:23:25.890] Placeholder Images[Split Arg] Processing complete
[15:23:25.891] Placeholder Images[Split Arg] Passing output '300 300' to Arg and Vars
[15:23:25.891] Placeholder Images[Arg and Vars] Processing complete
[15:23:25.892] Placeholder Images[Arg and Vars] Passing output 'https://picsum.photos/300 300/.jpg' to Conditional
[15:23:25.892] Placeholder Images[Conditional] Processing complete
[15:23:25.892] Placeholder Images[Conditional] Passing output 'https://picsum.photos/300 300/.jpg' to Post Notification
[15:23:25.893] Placeholder Images[Post Notification] Processing complete
[15:23:25.893] Placeholder Images[Post Notification] Passing output 'https://picsum.photos/300 300/.jpg' to Play Sound
[15:23:25.936] Placeholder Images[Arg and Vars] Passing output 'https://picsum.photos/300 300/.jpg' to Debug
[15:23:25.941] Placeholder Images[Debug] Interpreting given values as:
 Width: 300 300
Height:
```

### More details

If i set the separator to a space it works, but it doesn't work with the config

**Comment by learn2reid** [comment]
would also be nice to have a fallback to set one number & if second size isn't set, it uses the first size for both. please, thank you very much for your great work 🙏

**Comment by vitorgalvao** [comment]
Thank you for the report and the request. Fixed the bug and added the feature in 2024.3.
